documentation/src/main/asciidoc/topics/proc_using_mvn_archetypes.adoc

```diff
@@ -0,0 +1,58 @@
+[id='mvn_archetypes']
+= Creating {brandname} Projects
+{brandname} provides a set of Maven archetypes that you can use to generate
+skeleton projects with sample code and `pom.xml` files. You can then import
+these projects into your IDE and start building and running applications with
+{brandname} capabilities.
+
+.Prerequisites
+
+Before you can use the {brandname} archetypes, you need to do the following:
```

Comment:
this is incorrect. 
you don't need to compile the Infinispan project. The archetypes will be available in maven central.